ckanext/subscribe/fanstatic/subscribe.css

```diff
@@ -47,3 +47,7 @@ table#subscriptions {
 #subscriptions select#frequency {
     width: 120px;
 }
+
+label::after {
+    content: "";
```

Comment:
Why is this css needed? The colon should be removed from the text instead of the css.